yt/frontends/parthenon/data_structures.py

```diff
@@ -9,15 +9,15 @@
 from yt.data_objects.static_output import Dataset
 from yt.fields.magnetic_field import get_magnetic_normalization
 from yt.funcs import mylog
+from yt.geometry.geometry_enum import Geometry
```

Comment:
Following how it's done in the rest of the code base
```suggestion
from yt.geometry.api import Geometry
```